tests/python/unittest/test_tvmscript_error_report.py

```diff
@@ -144,6 +144,197 @@ def test_no_body():
     check_error(no_body, 3)
 
 
+def allocate_with_buffers() -> None:
```

Comment:
nit: why some tests have the return annotations but others haven't? IMHO, it's fine to ignore the type annotation of returning None.

Reply:
Please note that there are two types of functions here: 1. script function which will be parsed and 2. Python function with `test_`prefix. The first ones need type annotation while it is optional for the second ones

Reply:
Yeah I noticed that the ones with type annotations are TVM scripts. So just curious, does that mean when users are writing a TVM script, they have to specify type annotation?